app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
+<script type = "text/javascript" >
+	 $('#dodo').DataTable({
```

Comment:
> as there was as such no use of datables in that.
What do you mean there was no use of datatables in that ?! That is the parent template that shows this event table ... !